docs/operator-config.asciidoc

```diff
@@ -6,53 +6,56 @@ endif::[]
-|`metrics-port` |int |0 |Port to use for exposing metrics in the Prometheus format. Set 0 to disable
-|`operator-roles` |[]string |all |Roles this operator should assume. Valid values are namespace, global, webhook or all. Accepts multiple comma separated values.
-|`namespaces` |[]string |`""` |Namespaces in which this operator should manage resources. Accepts multiple comma-separated values. Defaults to all namespaces if empty or unspecified.
-|`ca-cert-validity` |duration (string) |1y |Duration representing how long before a newly created CA cert expires
-|`ca-cert-rotate-before` |duration (string) |1d |Duration representing how long before expiration CA certificates should be reissued
-|`cert-validity` |duration (string) |1y |Duration representing how long before a newly created TLS certificate expires
-|`cert-rotate-before` |duration (string) |1d |Duration representing how long before expiration TLS certificates should be reissued
-|`manage-webhook-certs` |bool |true |Enables automatic webhook certificate management
-|`operator-namespace` |string |`""` |K8s namespace the operator runs in
-|`webhook-secret` |string |`""` |K8s secret name mounted into /tmp/cert to be used for webhook certificates
-|`webhook-pods-label` |string |`""` |K8s label to select pods running the operator
-|`development` |bool |false |Enable developmenet mode. Only available as a CLI flag, not an environment variable
-|`debug-http-listen` |string |localhost:6060 |Listen address for the debug HTTP server. Only available in development mode
-|`auto-port-forward` |bool |false |Enables automatic port forwarding to allow running the operator outside the cluster. For dev use only as it exposes k8s resources on ephemeral ports to localhost
-|==========================
 
+Unless noted otherwise, environment variables can be used instead of flags to configure the operator as well. Simply convert the flag name to upper case and replace any dashes (`-`) with underscores (`_`). For example, the `enable-debug-logs` flag can be set by an environment variable named `ENABLE_DEBUG_LOGS`.
 
-Many of these should not need to be modified, but if necessary you can modify the operator YAML. For instance, to enable `DEBUG` level logs on the operator, restart it with the flag `--enable-debug-logs=true` like so:
+Duration values should be specified as numeric values suffixed by the time unit. For example, a duration of 10 hours should be specified as `10h`. Acceptable time unit suffixes are:
 
-[source,sh]
-----
-kubectl edit statefulset.apps -n elastic-system elastic-operator
-----
+[cols="m,d",options="header"]
+|===
+|Suffix |Unit
+|ms | Milliseconds
+|s  | Seconds
+|m  | Minutes
+|h  | Hours
+|===
 
-and change the following lines from:
 
-[source,yaml]
+Edit the operator deployment to change any of the flag values. For example, to enable `DEBUG` level logs, the operator needs to be restarted with the `--enable-debug-logs` flag set to `true`. This can be done as follows:
```

Comment:
I think we can clarify that it just needs to be edited and then the sset controller will handle restarting